### PR TITLE
[18.0][MIG] partner_address_street3: Migration to 18.0

### PR DESCRIPTION
**Pull request by LucasTran380381**
Ref:  [partner_address_street3]`(https://github.com/OCA/partner-contact/issues/1856)`

### CODE REVIEW

**Comment by nilshamerlinck** [comment]
ok, commit name should be `[MIG] partner_address_street3: Migration to 18.0` too
anything noteworthy to add in the PR description to help reviewer's life? (e.g. about the view changes)

**Comment by nilshamerlinck** [comment]
you can add yourself to `CONTRIBUTORS.md` and then announce your PR in `#trobz-oca-migrations` to get a review from a peer

**Comment by nilshamerlinck** [comment]
in terms of commits, I think you squashed a bit too much: you should keep `pre-commit auto fixes` commits separated from migration commits

File: partner_address_street3/views/res_partner.xml
Comment (LucasTran380381) [inline partner_address_street3/views/res_partner.xml:1]: change inherit field parent due to removed in v18

File: partner_address_street3/views/res_partner.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 />
             </xpath>
             <xpath
-                expr="/form//field[@name='child_ids']/kanban/field[@name='street2']"
```
Comment (nilshamerlinck) [inline partner_address_street3/views/res_partner.xml]: if street2 has been removed from that view, maybe adding street3 is not needed at all? can you find the odoo commit of that change to understand the rationale? might have been to save space

**Pull request closed without merging**